CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
 
 ## 1.21.9 (Not yet released)
 * Use the base filename instead of 'ClassName' in ctor snippet (PR:[#3385](https://github.com/OmniSharp/omnisharp-vscode/pull/3385))
+* Updated Razor
```

Comment:
@danroth27 I took the approach of a simple changelog. Thoughts?

Reply:
I dig it!